ml-agents/mlagents/trainers/ppo/trainer.py

```diff
@@ -51,35 +50,32 @@ def __init__(
             "use_recurrent",
             "summary_path",
             "memory_size",
-            "use_curiosity",
-            "curiosity_strength",
-            "curiosity_enc_size",
             "model_path",
+            "reward_signals",
         ]
-
         self.check_param_keys()
-        self.use_curiosity = bool(trainer_parameters["use_curiosity"])
+
+        # Make sure we have at least one reward_signal
+        if not self.trainer_parameters["reward_signals"]:
+            raise UnityTrainerException(
+                "No reward signals were defined. At least one must be used with the PPO trainer."
```

Comment:
Use the class name, do not hard code PPO here ?

Reply:
Fixed. 

BTW, I plan to make this even more generic, e.g. for printing the trainer_parameters, I'd move it to the base Trainer class and use the class name from there. Probably will come when we add SAC - with this PR I'm trying not to modify too much